src/server/terrain/mapgen/MapGenV1.zig

```diff
@@ -67,7 +67,7 @@ pub fn generateMapFragment(map: *MapFragment, worldSeed: u64) void {
 	// A ridgid noise map to generate interesting mountains.
 	const mountainMap = Array2D(f32).init(main.stackAllocator, scaledSize, scaledSize);
 	defer mountainMap.deinit(main.stackAllocator);
-	RandomlyWeightedFractalNoise.generateSparseFractalTerrain(map.pos.wx, map.pos.wy, 64, worldSeed ^ 6758947592930535, mountainMap, map.pos.voxelSize);
+	RandomlyWeightedFractalNoise.generateSparseFractalTerrain(map.pos.wx, map.pos.wy, 128, worldSeed ^ 6758947592930535, mountainMap, map.pos.voxelSize);
```

Comment:
This change, as well as the new biomes, change the terrain generation.
This breaks my current world in a bad way:
![Screenshot at 2024-06-03 11-32-48](https://github.com/PixelGuys/Cubyz/assets/43880493/cfcae9bc-0ff0-4557-a6ac-3b6dca763399)
![Screenshot at 2024-06-03 11-33-45](https://github.com/PixelGuys/Cubyz/assets/43880493/7072de65-06e1-425c-acba-4d6adaf4015d)
To prevent this, we will have to wait on #360 #350 and #349 before merging this pull request.